components/bl-autocomplete-component/src/index.js

```diff
@@ -11,7 +11,7 @@ export default function AutocompleteComponent({ component, eventHandlers }) {
 
   const rootRef = useRef();
   const autocompleteRef = useRef();
-  const [optionsList, setOptionsList] = useState([]);
+  const [optionsList, setOptionsList] = useState(validate(options));
```

Comment:
`useState(() => validate(options))` in order to do not run it on each render

Reply:
and what if the list is changing in runtime?

for instance:
- we pass a valid list
- then in some time we assign a new `options` to the component

Will we valid the new options?

Reply:
perhaps `useMemo` fits better for that, what do you think?